.changeset/eighty-dolphins-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-html-vas-response": patch
+---
+
+html-vas-response: add option for discrete scale points
```

Comment:
```suggestion
Add option for discrete scale points
```
No need to mention `html-vas-response` here since what you write here only appears in the changelog and release notes of `html-vas-response` :slightly_smiling_face: